Surface_mesh_simplification/doc/Surface_mesh_simplification/CGAL/Surface_mesh_simplification/Policies/Edge_collapse/Bounded_normal_change_filter.h

```diff
@@ -0,0 +1,54 @@
+
+namespace CGAL {
+namespace Surface_mesh_simplification {
+
+/*!
+\ingroup PkgSurfaceMeshSimplificationRef
+
+The class `Bounded_normal_change_filter` is a model for the `GetFilter` concept.
```

Comment:
Strange that both the class and the template parameter are model of `GetFilter`

Reply:
Isn't that similar to vertex classes in the triangulation packages.

Reply:
Changed to `Filter`.